### ISSUE TITLE
Issues downloding data

### ISSUE THREAD

**Issue by SaraAV** [ropensci/rnoaa#397]
<details> <summary><strong>Session Info</strong></summary>

```r

```
</details>

**Comment by SaraAV** [comment]
Hello,

I now this is not a bug in the code, but I am really struggling to download data using the ncdc function. Before, it was working but at some point I got these errors:

**Failed to connect to www.ncdc.noaa.gov port 443: Timed out

Timeout was reached: [www.ncdc.noaa.gov] Operation timed out after 10010 milliseconds with 0 out of 0 bytes received**

Anyone could give some hints to solve these issues? I am connected to the internet!

Thank you!

**Comment by sckott** [comment]
Hi, It's probably a temporary problem with NOAA. I'd try again at least a few hours later and see if you have the same problem

**Comment by SaraAV** [comment]
Hi sckott,

The three types of errors, are related to the same type of issue? the server of NOAA?

Thank you!

**Comment by sckott** [comment]
Yeah, they're different errors of course, but related. Have you tried again recently? Just tried this now and it worked for me

```r
ncdc(datasetid='GHCND', locationid = 'FIPS:02', startdate = '2010-05-01',
        enddate = '2010-05-10')
```
